components/Projects/ProjectCard/ProjectCard.tsx

```diff
@@ -0,0 +1,93 @@
+import ProjectCardDescription from './ProjectCardDescription';
+import GithubButton from './GithubButton';
+import DiscordButton from './DiscordButton';
+import { useMemo } from 'react';
+import ProjectImagePlaceholder from './ProjectImagePlaceholder.png';
+import ImageWithFallback from '@/components/utils/ImageWithFallback';
+import TagList from './TagList';
+import AvatarList from './AvatarList';
+import { AvatarData } from './Avatar';
+
+export interface ProjectCardProps {
+  projectThumbnailSrc: string;
+  updatedDate: Date;
+  createdDate: Date;
+  projectName: string;
+  contributorCount: number;
+  contributorAvatars: AvatarData[];
+  description: string;
+  tags: string[];
+  githubLink?: string;
+  discordLink?: string;
+}
+
+export default function ProjectCard({
+  projectThumbnailSrc,
+  updatedDate,
+  createdDate,
+  projectName,
+  contributorCount,
+  contributorAvatars,
+  description,
+  tags,
+  githubLink,
+  discordLink,
+}: ProjectCardProps) {
+  const updatedDateString = useMemo(
```

Comment:
Do we really need `useMemo` here?
If you drop it and just -> 
`const updatedDateString = updatedDate.toLocaleDateString('he-IL').replaceAll('.', '/')`
What is the difference?